ros_ws/src/navigation_stack/navigation_stack_config/config/amcl_params.yaml

```diff
@@ -4,19 +4,19 @@ odom_frame_id: "odom"
 #base_frame_id: "base_footprint"
 global_frame_id: "map"
 
-odom_model_type: "diff"
-odom_alpha5: 0.1
+odom_model_type: "diff-corrected"
```

Comment:
These changes aren't really necessary because we aren't using AMCL anymore but I guess it's still good to include them.